keps/sig-auth/4633-anonymous-auth-configurable-endpoints/README.md

```diff
@@ -587,6 +589,9 @@ in back-to-back releases.
 - Feature implemented behind a feature flag
 - Full unit and integration test coverage
 
+#### Beta
```

Comment:
The unit and integration tests sections are missing the links to the tests

Reply:
Done. Thanks for pointing that out!